cds/config.py

```diff
@@ -740,6 +741,53 @@ def _(x):
     }
 ]
 
+FRONTPAGE_TREND_TOPICS = [
+    {
+        'label': 'antimatter',
+        'qs': 'keyword=antimatter',
+    },
+    {
+        'label': 'dark matter',
+        'qs': 'q=keywords.name:"dark matter"',
+    },
+    {
+        'label': 'higgs',
+        'qs': 'keyword=higgs',
+    },
+    {
+        'label': 'HL-LHC',
+        'qs': 'q=keywords.name:"HL-LHC"',
+    },
+    {
+        'label': 'LHC',
+        'qs': 'q=keywords.name:"LHC"',
+    },
+    {
+        'label': 'CLOUD',
+        'qs': 'keyword=CLOUD',
+    },
+    {
+        'label': 'FCC',
+        'qs': 'keyword=FCC',
+    },
+    {
+        'label': 'AWAKE',
+        'qs': 'keyword=AWAKE',
+    },
+    {
+        'label': 'collisions',
+        'qs': 'keyword=collisions',
+    },
+    {
+        'label': 'history',
+        'qs': 'keyword=history',
+    },
+    {
+        'label': 'video new releases',
+        'qs': 'q=',
```

Comment:
I am not sure what they want to achieve with this last one, this is last videos uploaded but we have already this in the home page ....